### DISCUSSION TITLE
Optionally display `belongs_to` associations like `has_one` associations.

### DISCUSSION THREAD

**Discussion by calebuharrison** [avo-hq/avo#398] [Feature requests]
`has_one` resource associations render almost the entire "show" view on the page of the associated resource. It would be great if `belongs_to` associations could receive the same treatment.

**Comment by adrianthedev** [comment]
Yes, this is a good idea. Will move it up on the roadmap.

**Reply by calebuharrison** [reply]
@adrianthedev One more detail concerning this. Currently, the `belongs_to` field on the `edit` page is a simple dropdown that displays the configured `title` of each resource. Perhaps this will change once they're treated like `has_one` associations, but if not, it would be great if either:

1. Clicking that dropdown triggered the search bar, from which the desired resource could be selected
2. The value of each dropdown option were configurable, because often the lone `title` of a resource is not unique

**Reply by adrianthedev** [reply]
Yup. Good feedback. And make sense.

1. Have a dynamic/fuzzy select if there are a lot of records.
2. Like on [search](https://docs.avohq.io/1.0/search.html#configure-the-search-result), customize the label of the result

**Reply by adrianthedev** [reply]
BTW, regarding 2. you could have a [computed field](https://github.com/avo-hq/avo/pull/402/files#diff-b30f6403c4bf527659f7ab9869c1074349bbe2c54254cd935f92fa5ce68b97b9R5) for that `title` [attribute](https://github.com/avo-hq/avo/pull/402/files#diff-e0063e82293acfb0f1e2ed4400cd0352a0941828082d6e10041738228a5ad599R2).

**Reply by calebuharrison** [reply]
Well, I may not necessarily want the title of the resource to change elsewhere. In my opinion, though, option 1 is the way to go.

**Reply by calebuharrison** [reply]
@adrianthedev The dynamic/fuzzy select is actually the most important part of this feature request to me. Should I split that out as a separate request?

**Reply by adrianthedev** [reply]
Yes, I guess that would be best.

And this ticket remains just for the display feature (`belongs_to` like`has_one`)?

**Reply by calebuharrison** [reply]
Okay, I'll create a new one!
